packages/dev/core/src/Layers/layer.ts

```diff
@@ -35,6 +35,20 @@ export class Layer {
      */
     public isBackground: boolean;
 
+    private _applyPostProcess: boolean = true;
+    /**
+     * If scene post processes should affect this Layer.
+     * Is always true if the layer is a background
+     * layer (this.isBackground === false), as in the case
+     * it will always be drawn before post processes.
+     */
+    public set applyPostProcess(value: boolean) {
+        this._applyPostProcess = value;
+    }
+    public get applyPostProcess(): boolean {
+        return this.isBackground || this._applyPostProcess;

```

Comment:
I do not get why there is the this.isBackground here actually or we should use && instead?
Because then whatever I use in applyPostProcess, if isBackground is true then we will apply the postprocess

Then what is the point of this feature? I thought it was to turn the applyPostprocess off but here it will not work

Reply:
oh wait! No, ok I understood....
This is a bit of a spaghetti code but now I get it ;D